agents/host-amd64/setup.sh

```diff
@@ -16,9 +16,10 @@
 
 SERVICE_NAME=opsverse-agent
 SERVICE_FILE=/etc/systemd/system/${SERVICE_NAME}.service
+SYSV_SERVICE_FILE=/etc/init.d/${SERVICE_NAME}
 
 NODE_EXPORTER_SERVICE_NAME=node_exporter
-NODE_EXPORTER_SERVICE_FILE=/etc/systemd/system/${NODE_EXPORTER_SERVICE_NAME}.service
```

Comment:
I see NODE_EXPORTER_SERVICE_FILE referred later

Reply:
GREAT CATCH - i meant to copy this line, not delete it.

Fixed and tested on systemd machine.
![Thu 19 May 2022 06-28-18 PM PDT](https://user-images.githubusercontent.com/88401519/169430499-22362e15-e18e-46da-a9e6-4de9918b87d4.png)